app/frontend/src/components/Markdown.tsx

```diff
@@ -27,7 +27,7 @@ const useStyles = makeStyles((theme) => ({
     "& h4": theme.typography.h4,
     "& h5": theme.typography.h5,
     "& h6": theme.typography.h6,
-    "& p": theme.typography.body1,
+    "& p": { ...theme.typography.body1, margin: 0 },
```

Comment:
On second thought, it's better with the paragraph margin. Why did you want to get rid of it in the local info?

Reply:
I thought it made the gap between the paragraph and the title a little too big, and the fact it's closer to the moderators section makes it looks like it belongs more to that section rather than the "general information" heading immediately below.

**Without margin (current code)**
![Screenshot 2021-06-06 at 16 18 16](https://user-images.githubusercontent.com/13669362/120929910-d85fd600-c6e2-11eb-949f-a2a52a353e24.png)

**With margin**
![Screenshot 2021-06-06 at 16 18 23](https://user-images.githubusercontent.com/13669362/120929912-dc8bf380-c6e2-11eb-9743-a6e35e208297.png)

Reply:
Maybe a better solution is to push the "community moderators" further down? Removing this margin globally does make it a bit too close to the back button on discussion pages so hmm maybe that's the way to go

Reply:
I don't mind either way for the titles, but I'm thinking about when you have two paragraphs together, there should be some spacing. If it's set to 0, then it's equivalent to `<br>` and it shouldn't be

Reply:
@darrenvong ☝️